modules/microshift-gitops-debug.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-gitops-debug_{context}"]
+= Debugging GitOps
+
+There are two ways to debug GitOps in your {microshift-short} instance: using an sos report and using the {oc-first}.
+
+.Prerequisites
+
+* The `sos` package is installed.
+* The `oc` command line tool is installed.
+
+.Procedure
+
+. Using sos reports to debug GitOps.
```

Comment:
I think it could be useful to add a link about how to use sos report as reference: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/getting_the_most_from_your_support_experience/generating-an-sos-report-for-technical-support_getting-the-most-from-your-support-experience Not sure if this is the best info